docs/using.rst

```diff
@@ -42,126 +22,35 @@ above method instead.
    https://github.com/letsencrypt/letsencrypt/archive/master.zip
 
 
-.. _prerequisites:
```

Comment:
this reflink is still referenced in other places in the documentation. It should not be removed